run.py

```diff
@@ -1,4 +1,4 @@
 from api import create_app
 
 if __name__ == '__main__':
-    create_app().run(debug=False)
+    create_app().run(debug=True)
```

Comment:
Why we need to change this ?

Reply:
debug=True offers me debugger errors when accessing an endpoint. Otherwise it just says Internal Server Error.